docs/getting_started/configuration.rst

```diff
@@ -279,7 +279,20 @@ Otherwise, the controller won't be able to properly compensate for the end effec
 
     End effector variants supported by Trossen Robotics are provided in :class:`trossen_arm::StandardEndEffector`.
 
-.. note:: Currently, only rack-and-pinion end effectors are supported.
+.. tip::
+
+    New in version 1.8.3: the original rack-and-pinion end effector can be replaced with a custom end effector, e.g. :member:`trossen_arm::StandardEndEffector::no_gripper`.
```

Comment:
I would mention that the gripper assembly can be removed in favor of a custom end effector or no end effector at all. Mentioning the no_gripper struct does not accurately describe this in whole.